lacommunaute/utils/templatetags/theme_inclusion.py

```diff
@@ -75,11 +75,11 @@ def import_static_CSS_theme_inclusion():
     scripts_import = ""
     for css_dep in CSS_DEPENDENCIES_THEME:
         if css_dep["is_external"]:
-            scripts_import += (
-                '<link rel="stylesheet" href="{}" integrity="{}" ' 'crossorigin="anonymous" type="text/css">'
-            ).format(css_dep["src"], css_dep["integrity"])
+            scripts_import += ('<link rel="stylesheet" href="{}" integrity="{}" ' 'crossorigin="anonymous">').format(
+                css_dep["src"], css_dep["integrity"]
+            )
         else:
-            scripts_import += '<link rel="stylesheet" href="{}" type="text/css">'.format(static_theme(css_dep["src"]))
+            scripts_import += '<link rel="stylesheet" href="{}">'.format(static_theme(css_dep["src"]))
```

Comment:
Oui c'est vrai qu'on peut omettre type="text/css". Moi j'ai l'habitude de laisser, mais why not

Reply:
MDN dit :
> The common use of this attribute is to define the type of stylesheet being referenced (such as text/css), but given that CSS is the only stylesheet language used on the web, not only is it possible to omit the type attribute, but is actually now recommended practice.